static-environments/eslint-e2e-env/project.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "eslint-e2e-env",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "static-environments/eslint-e2e-env/src",
+  "projectType": "library",
+  "targets": {},
+  "tags": ["scope:internal", "type:feature"]
```

Comment:
Tagging is `type:feature` doesn't seem appropriate, would go with something more limiting like `type:util`. But better still would be to not make this an Nx project at all - see previous comment.

Reply:
removed